Spigot-Server-Patches/0252-Prevent-logins-from-being-processed-when-the-player-.patch

```diff
@@ -0,0 +1,37 @@
+From b7eb6f7291b81400eeaf3b9702c52927f66080d3 Mon Sep 17 00:00:00 2001
+From: killme <killme-git@ibts.me>
+Date: Sun, 12 Nov 2017 19:40:01 +0100
+Subject: [PATCH] Prevent logins from being processed when the player has
+ disconnected
+
+
+diff --git a/src/main/java/net/minecraft/server/LoginListener.java b/src/main/java/net/minecraft/server/LoginListener.java
+index 75df928..e98c721 100644
+--- a/src/main/java/net/minecraft/server/LoginListener.java
++++ b/src/main/java/net/minecraft/server/LoginListener.java
+@@ -60,7 +60,12 @@ public class LoginListener implements PacketLoginInListener, ITickable {
+         }
+         // Paper end
+         if (this.g == LoginListener.EnumProtocolState.READY_TO_ACCEPT) {
+-            this.b();
++            // Paper start - prevent logins to be processed even though disconnect was called
++            if (!networkManager.isOpen()) {
```

Comment:
The joys of commenting when tired, this shouldn't actually be negated, and would probably be fine to move into the if statement itself to create a bit of a smaller diff

Reply:
and on the compile error, `isConnected()` is apparently what I should have referred to in my sleep deprived state

Reply:
Fixed